docs/man/encrypt/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Encrypt file or stdin as a TDF
+command:
+  name: encrypt [file]
+  flags:
+    - name: out
+      shorthand: o
+      description: A filename and extension that will be TDFd (i.e. '-o password.txt' -> 'password.txt.tdf', default 'sensitive.txt.tdf' or <file>.tdf) and placed in the current working directory.
```

Comment:
I don't think this should be the default behavior. I would expect the default behavior is output to stdout. So we can support something like this

```
tar cv /secrets | gzip | otdfctl encrypt | s3 cp - s3://bucket/secrets.tar.gz.tdf
```

Reply:
Great feedback. Updated in newer commits.